images/router/f5/Dockerfile

```diff
@@ -1,11 +1,8 @@
 #
-# This is the HAProxy router for OpenShift Origin.
+# This is the F5 router for OpenShift Origin.
 #
 # The standard name for this image is openshift/origin-f5-router
 #
-FROM openshift/origin-base
+FROM openshift/origin
 
-ADD bin/openshift /usr/bin/openshift
-RUN ln -s /usr/bin/openshift /usr/bin/openshift-router
-
-ENTRYPOINT ["/usr/bin/openshift-router", "--type=f5"]
+ENTRYPOINT ["/usr/bin/openshift-f5-router"]
```

Comment:
Hrm - all of our symlinks for openshift/origin are in the base image - I'd
rather have it there.  The router image is distinct enough that I don't
think it's considered the same.  If we switched the f5 to a separate binary
we'd want to, but for now I'd rather keep them all together.

On Mon, Aug 31, 2015 at 3:44 PM, Ram Ranganathan <notifications@github.com>
wrote:

> In images/router/f5/Dockerfile
> <https://github.com/openshift/origin/pull/4428#discussion_r38344729>:
>
> >
> > -ADD bin/openshift /usr/bin/openshift
> > -RUN ln -s /usr/bin/openshift /usr/bin/openshift-router
> > -
> > -ENTRYPOINT ["/usr/bin/openshift-router", "--type=f5"]
> > +ENTRYPOINT ["/usr/bin/openshift-f5-router"]
>
> [image: :+1:] for a cleaner dockerfile - but see above comment on moving
> the symlink for openshift-f5-router.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/4428/files#r38344729>.
>



-- 
Clayton Coleman | Lead Engineer, OpenShift